### PR TITLE
#3 scalac option changes in Scala 2.13.3

### PR DESCRIPTION
**Pull request by mr-git**

### CODE REVIEW

File: src/sbt-test/sbt-scalac-opts-plugin/scalacOptions/build.sbt [hunk 1]
```diff
@@ -1,3 +1,10 @@
 crossScalaVersions := Seq(
   "2.12.8",
-  "2.13.0-RC1")
+  "2.12.9",
+  "2.12.10",
+  "2.12.11",
+  "2.13.0",
+  "2.13.1",
+  "2.13.2",
+  "2.13.3"
```
Comment (t3hnar) [inline src/sbt-test/sbt-scalac-opts-plugin/scalacOptions/build.sbt:3]: why do we need so many cross versions?

Comment (mr-git) [reply src/sbt-test/sbt-scalac-opts-plugin/scalacOptions/build.sbt:3]: that is for internal test project, but I can leave only latest two

Comment (t3hnar) [reply src/sbt-test/sbt-scalac-opts-plugin/scalacOptions/build.sbt:3]: yes please,  either last two or convince that "your use case" is usable for others as well :)

Comment (mr-git) [reply src/sbt-test/sbt-scalac-opts-plugin/scalacOptions/build.sbt:3]: I experimented and expected build failure on 2.13.3 due to `-Xlint`, but it didn't happen, only error message in log, hard to spot - removed useless targets

**Pull request merged**